### PR TITLE
QoS QueueSort: rename files&plugin and fix a comment

### PR DESCRIPTION
**Pull request by yuanchen8911**
This PR made the following changes to QoS QueueSort plugin by following the kubernetes convention.
  -  File name: queuesort.go -> queue_sort.go
  -  File name: queuesort_test.go -> queue_sort_test.go
  -  Plugin name: QoSSort -> QOSSort
  -  Fixed the comment of Less()

### CODE REVIEW

**Comment by yuanchen8911** [comment]
The same failed integration test as coscheduling. Did I miss to change something?  Testing in our local clusters worked just fine.

File: pkg/qos/queue_sort.go [hunk 1]
```diff
@@ -25,22 +25,22 @@ import (
 )
 
 // Name is the name of the plugin used in the plugin registry and configurations.
-const Name = "QoSSort"
+const Name = "QOSSort"
 
-// QoSSort is a plugin that implements QoS class based sorting.
-type Sort struct{}
+// QOSSort is a plugin that implements QoS class based sorting.
+type QOSSort struct{}
```
Comment (Huang-Wei) [inline pkg/qos/queue_sort.go:31]: It's sometimes conventional to not include the package name (qos) in the type name. Ref: https://github.com/golang/go/wiki/CodeReviewComments#package-names.

I would suggest to keep it as, or later renamed to `GenericSort`.

**Comment by Huang-Wei** [comment]
> The same failed integration test as coscheduling. Did I miss to change something? Testing in our local clusters worked just fine.

I created issue #19 to track this.

File: pkg/qos/queue_sort.go (hunk 1 above)
Comment (yuanchen8911) [reply pkg/qos/queue_sort.go]: > It's sometimes conventional to not include the package name (qos) in the type name. Ref: https://github.com/golang/go/wiki/CodeReviewComments#package-names.
> 
> I would suggest to keep it as, or later renamed to `GenericSort`.
Agree, changed it back to Sort.

Comment (yuanchen8911) [reply pkg/qos/queue_sort.go]: If QueueSort is just one of plugins in package "qos" and other QoS related plugins can be added to the package in the future, the package name "qos" makes sense.  I'll submit a PR on the GenericSort plugin based on our discussions.  Thanks.

Comment (yuanchen8911) [reply pkg/qos/queue_sort.go]: > > The same failed integration test as coscheduling. Did I miss to change something? Testing in our local clusters worked just fine.
> 
> I created issue #19 to track this.

After reverting the QueueSort plugin struct name to "Sort", it passed the integration test.

**Comment by yuanchen8911** [comment]
@Huang-Wei The changes are straightforward and  minor. Any comments or concerns?  The same thing for the coscheduling PR. After they are merged, we will then look at more significant changes as discussed. Thanks.

**Comment by Huang-Wei** [comment]
Thanks.

/lgtm
/approve

**Pull request merged**